### DISCUSSION TITLE
Git-lfs

### DISCUSSION THREAD

**Discussion by m-pawar** [git-lfs/git-lfs#4823] [General]
How to download repositories by using git-lfs?
Like how do I download this by using Git-Lfs, **https://github.com/Unity-Technologies/FPSSample**???

**Comment by bk2204** [comment]
Hey,

If you have Git LFS installed on your system, run `git lfs install` once on your system, and clone the repository.  The LFS files will be checked out automatically.  If you've already cloned it but haven't run `git lfs install`, run that command first, then run `git lfs pull` and the LFS files will be checked out.
